core/cap-0035.md

```diff
@@ -437,7 +437,7 @@ operation.
 
 Possible return values for the `ClawbackOp` are:
 - `CLAWBACK_SUCCESS` if the clawback is successful.
-- `CLAWBACK_MALFORMED` if the `asset` value is malformed or `amount` is < 1.

```

Comment:
I'm not sure if this is right - will XDR catch an invalid asset code with a non-trailing null byte or extended ascii chars?

Reply:
You're right. Forgot about this!

Reply:
Reverted in https://github.com/stellar/stellar-protocol/pull/829/commits/0dc84dd6df7f58c49be1d4e0f2c7338626ae0e14.